package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "tape test/*.test.js",
```

Comment:
Experimented testing the server using supertest but encountered difficulty making this compatible with tape for injecting and then stopping the server. For now have set up basic tape testing without a server example.